gallery/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicatorPage.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxVulnerabilityIndicatorExample from './NxVulnerabilityIndicatorExample';
+import NxVulnerabilityIndicatorColumnExample from './NxVulnerabilityIndicatorColumnExample';
+import NxVulnerabilityIndicatorGridExample from './NxVulnerabilityIndicatorGridExample';
+
+const nxVulnerabilityIndicatorCode = require('./NxVulnerabilityIndicatorExample?raw'),
+    nxVulnerabilityIndicatorColumnCode = require('./NxVulnerabilityIndicatorColumnExample?raw'),
+    nxVulnerabilityIndicatorGridCode = require('./NxVulnerabilityIndicatorGridExample?raw');
+
+const NxVulnerabilityIndicatorPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        Basic style for small indicator which displays a number representing a count of violations, and a short pre-set
```

Comment:
The component is currently named nx _vulnerability_ indicator, but this sentence talks about _violations_?  Which is it intended to work over?  IMO both, as well any anything else that could be categorized in this way.  I'd propose a different name for the component to reflect this: `NxThreatCounts`

Reply:
> The component is currently named nx _vulnerability_ indicator, but this sentence talks about _violations_? Which is it intended to work over? IMO both, as well any anything else that could be categorized in this way. I'd propose a different name for the component to reflect this: `NxThreatCounts`

https://github.com/sonatype/sonatype-react-shared-components/pull/343/commits/bb302d60abf5b3fb11308011c6e1f612dc76c1a1